src/views/MapView.vue

```diff
@@ -1,54 +1,53 @@
 <template>
   <div>
-    <div class="header" v-if="showHeader">
```

Comment:
@florisdobber / @willmt80 Seems like there are a lot of changes in this file, are these all what we want? Or did something get messed up from this PR sitting around for so long?

Reply:
Most of the changes are pretty self contained, there are some name changes on an open PR but merging should be pretty easy.